spec/factories/subjects.rb

```diff
@@ -125,4 +119,15 @@
       type { :SecondarySubject }
     end
   end
+
+  factory :secondary_subject do
```

Comment:
does this collide with the traits above...

Reply:
No, because you need the factory name with a trait:

```ruby
create(:subject, :modern_languages)
```

vs

```ruby
create(:secondary_subject, :modern_languages)
```

Reply:
the former would be the same as the latter...